crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -399,17 +393,25 @@ pub mod transaction {
     #[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
     #[serde(deny_unknown_fields)]
     pub struct DeployAccountTransaction {
-        #[serde_as(as = "TransactionVersionAsHexStr")]
-        pub version: TransactionVersion,
+        pub contract_address: ContractAddress,
```

Comment:
This `contract_address` property seems mandatory, but I can't find it in any schema related to DEPLOY or DEPLOY_ACCOUNT transactions. Is it an aggregate entity based on [Pending]Transaction and Receipt?

Reply:
Unfortunately there's no documentation for the REST response returned by the feeder gateway. We can mostly deduce the schema from a `get_block` or `get_transaction` response...